ush/generate_FV3LAM_wflow.py

```diff
@@ -290,240 +287,7 @@ def generate_FV3LAM_wflow(
         verbose=debug,
     )
     cp_vrfy(FIELD_DICT_IN_UWM_FP, FIELD_DICT_FP)
-    #
```

Comment:
Everything removed here is related to setting up the staged namelist file for the experiment directory. All the logic has been absorbed by the appropriate configuration YAMLs and is now handled in setup.py, like all other configuration settings.

I don't know why this was done so late in the first place.